kola/tests/kubeadm/templates.go

```diff
@@ -353,12 +355,22 @@ spec:
   calicoNetwork:
     # Note: The ipPools section cannot be modified post-install.
     ipPools:
-      - blockSize: 26
-        cidr: {{ .PodSubnet }}
-        encapsulation: VXLANCrossSubnet
-        natOutgoing: Enabled
-        nodeSelector: all()
+    - blockSize: 26
+      cidr: {{ .PodSubnet }}
+      encapsulation: VXLANCrossSubnet
+      natOutgoing: Enabled
+      nodeSelector: all()
   flexVolumePath: /opt/libexec/kubernetes/kubelet-plugins/volume/exec/
+
+---
+
+# This section configures the Calico API server.
```

Comment:
Is this section mandatory? Or it's just a placeholder for a later further customization?

Reply:
I dunno. It appeared in the custom resources manifest file in Calico between 3.19 and 3.20. I think it's best to have it to have our test calico setup as closest as possible to what upstream suggests in their quickstart guide.